docs/es/faq.md

```diff
@@ -82,7 +82,7 @@ estos navegadores, no dude en hacérnoslo saber en https-everywhere en EFF.org
 (pero tenga en cuenta que modificar document.location o window.location en
 JavaScript no es seguro).
 
-### [¿Por qué utilizar una lista blanca de sitios que admiten HTTPS? ¿Por qué no pueden intentar utilizar HTTPS para cada sitio, y sólo volver a HTTP si no está disponible?](#why-use-a-whitelist-of-sites-that-support-https-why-cant-you-try-to-use-https-for-every-last-site-and-only-fall-back-to-http-if-it-isnt-available)
+### [¿Por qué utilizar una lista blanca de sitios que admiten HTTPS? ¿Por qué no pueden intentar utilizar HTTPS para cada sitio, y sólo volver a HTTP si no está disponible?](#why-use-a-allowlist-of-sites-that-support-https-why-cant-you-try-to-use-https-for-every-last-site-and-only-fall-back-to-http-if-it-isnt-available)
```

Comment:
"lista blanca" still translates as "white list" literally.

Reply:
This is true, I am working internally to get a proper translation of this term since Spanish is a secondary language for me personally.

Reply:
¿Por qué utilizar una lista de sitios aprobados que admiten HTTPS?